well_records.rb

```diff
@@ -0,0 +1,183 @@
+require 'open-uri'
```

Comment:
Always use double quotes.

I won't mark every place in these two files

Reply:
Rspec generates files with single quotes, as rspec-snippet prints single quotes. Sorry, this happens automatically :)